metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -646,9 +646,6 @@ def _build_derived_metric_output_node(
             queried_agg_time_dimension_specs = queried_linkable_specs.included_agg_time_dimension_specs_for_metric(
                 metric_reference=metric_spec.reference, metric_lookup=self._metric_lookup
             )
-            assert (
```

Comment:
This error is handled in the `__post_init__` for `JoinToTimeSpineNode`, so I removed the duplication.